### PR TITLE
Advent 2015 Series: Hydra introduction

### PR DESCRIPTION
**Pull request by arekkas**
This is a draft for the Hydra introduction which is due 12/21/2015 (I believe).

### CODE REVIEW

**Comment by arekkas** [comment]
I have to re-run the tutorial on different machines but as far as the text goes it is good for :eyes:

File: content/advent-2015/hydra-auth.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
++++
+author = ["arekkas"]
+date = "2015-12-17T16:05:49+01:00"
```
Comment (mattfarina) [inline content/advent-2015/hydra-auth.md]: Can you update the date so it's on December 21st.

Comment (arekkas) [inline content/advent-2015/hydra-auth.md]: will do, eg `2015-12-21T00:00:00+01:00` ?

Comment (mattfarina) [inline content/advent-2015/hydra-auth.md]: I think that will work. Thanks.

Comment (arekkas) [inline content/advent-2015/hydra-auth.md]: Fixed it :)

**Comment by arekkas** [comment]
Ok I tested everything on Linux Mac and OSX, should be good to merge :)

**Comment by mattfarina** [comment]
Thanks. You're post is now up at https://blog.gopheracademy.com/advent-2015/hydra-auth/

**Comment by arekkas** [comment]
:open_hands:

**Pull request closed without merging**